src/components/header/filter/FilterDropdown.tsx

```diff
@@ -49,6 +49,10 @@ const Filter: React.FC<FilterDropdownProps> = ({}) => {
         filterText: '',
       },
     });
+    dispatch({
+      type: 'SET_PAGE',
+      payload: 1,
+    });
```

Comment:
this action is adding empty filter which doesn't trigger a filter action. Only after users set all required values for an filter item, then it will.

Reset page number here is not right.